tests/test_sticker.py

```diff
@@ -30,9 +30,8 @@
 
 @pytest.fixture(scope='function')
 def sticker_file():
-    f = open('tests/data/telegram.webp', 'rb')
-    yield f
-    f.close()
+    with open('tests/data/telegram.webp', 'rb') as f:
+        yield f
```

Comment:
if this is not really necessary for this PR, please revert - spares merging trouble :)